packages/aws-cdk-lib/aws-eks/lib/cluster.ts

```diff
@@ -1183,8 +1183,8 @@ abstract class ClusterBase extends Resource implements ICluster {
     if (!this._spotInterruptHandler) {
       this._spotInterruptHandler = this.addHelmChart('spot-interrupt-handler', {
         chart: 'aws-node-termination-handler',
-        version: '0.18.0',
-        repository: 'https://aws.github.io/eks-charts',
+        version: '0.27.0',
```

Comment:
We are having a discussion now.
https://github.com/aws/aws-cdk/issues/34217#issuecomment-2824842180